app/main.py

```diff
@@ -1,8 +1,66 @@
-class Car:
-    # write your code here
-    pass
-
-
-class CarWashStation:
-    # write your code here
-    pass
+class Car:
+    def __init__(
+            self,
+            comfort_class: float,
+            clean_mark: float,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+
+class CarWashStation:
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: float,
+            average_rating: float,
+            count_of_ratings: float
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (
+                (car.comfort_class * (self.clean_power - car.clean_mark)
+                 ) * (self.average_rating / self.distance_from_city_center)
+        )
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rating: float) -> None:
+        total_rating = self.average_rating * self.count_of_ratings
+        self.count_of_ratings += 1
+        self.average_rating = (total_rating + rating) / self.count_of_ratings
+        self.average_rating = round(self.average_rating, 1)
+
+
+bmw = Car(3, 3, "BMW")

```

Comment:
I see that you tested your code, but please add ONLY the solution to the task in the PR
Remove lines from 52 to 66
And also why did you deleted so many files ?) You should have only worked in the main file